src/main/steps/applicant/orders/template.njk

```diff
@@ -0,0 +1,34 @@
+{% extends "common/template.njk" %}
+
+{% block page_content %}
+  <h1 class="govuk-heading-xl">
+    <span class="govuk-caption-l">{{section}}</span>
+    {{title}}
+  </h1>
+  <div class="govuk-form-group">
+    <fieldset class="govuk-fieldset">
+      <p class="govuk-body">Case number #{{userCase.id}}</p>
+    </fieldset>
+    <hr class="govuk-section-break govuk-section-break govuk-section-break--visible">
+  </div>
+
+  <ol class="app-task-list">
+
+    <li>
+
+      <dl class="govuk-summary-list govuk-summary-list--no-border">
+        {% for file in userCase.orderCollection %}
```

Comment:
This logic to populate orders should be moved to utils.ts, as it can be reused for any other order population pages.